OracleSOASuite/dockerfiles/12.2.1.3/Dockerfile.patch

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (c) 2016-2017 Oracle and/or its affiliates. All rights reserved.
```

Comment:
It should  be "Copyright (c) 2017, 2020, Oracle and/or its affiliates."  as 2017 was the year this content was first published and 2020 is the year it was last updated. Also, we no longer include the phrase "All rights reserved" in our copyright statements.